src/app/components/hmcts-global-footer/hmcts-global-footer.component.html

```diff
@@ -7,17 +7,16 @@ <h2 class="hmcts-footer__heading">
           </h2>
           <ul class="govuk-footer__list">
               <li class="hmcts-footer__list-item">
-                  Email: <a class="hmcts-footer__link" href="mailto:{{help.email.address}}">{{help.email.text}}</a>
-              </li>
-              <li class="hmcts-footer__list-item">Phone: {{help.phone.text}}</li>
-              <li class="hmcts-footer__list-item">Opening times: {{help.opening.text}}</li>
-          </ul>
-          <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
-          <ul class="hmcts-footer__list hmcts-footer__list--inline">
-            <li class="hmcts-footer__list-item" *ngFor="let item of navigation.items">
-              <a class="govuk-footer__link" [routerLink]="item.href" [target]="item.target">{{item.text}}</a>
-            </li>
+                Contact your service representative for further help and information
+<!--                  Email: <a class="hmcts-footer__link" href="mailto:{{help.email.address}}">{{help.email.text}}</a>-->
+
           </ul>
+<!--          <hr class="govuk-section-break govuk-section-break&#45;&#45;l govuk-section-break&#45;&#45;visible">-->
```

Comment:
I understand this is commented out code, I also understand why we wouldn't want delete it, but why are the '-' s are replaced with '#45;'s here?

Reply:
sorry forgot to push the last commit